docs/guides/expectations/creating_custom_expectations/overview.md

```diff
@@ -1,18 +1,76 @@
 ---
-title: Creating custom Expectations
+title: Overview
 ---
-import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Prerequisites from './components/prerequisites.jsx'
 
+You can extend the functionality of Great Expectations by creating your own custom Expectations. You can also enrich Great Expectations as a shared standard for data quality by contributing new Expectations to the open source project.
 
-(Text goes here)
+Both processes follow the same streamlined steps. This section will teach you how.
```

Comment:
I'm not certain this is accurate.  It seems to me that creating a custom expectation is one process, and contributing a created expectation to the open source is another process that is dependent on the creation process having been completed. Possibly you mean that creating a custom expectation and creating and contributing a custom expectation follow the same steps -- but that's still a bit inaccurate since the second process would have additional steps that the first stopped before reaching.

I would change this to: These processes compliment each other and their steps are streamlined so that one flows into the other.  Once you have created a custom expectation it is simple to contribute it to the open source project, and this section will teach you how to do both.

Reply:
```suggestion
These processes compliment each other and their steps are streamlined so that one flows into the other. Once you have created a custom Expectation, it is simple to contribute it to the open source project. This section will teach you how to do both.
```